pkg/cmd/workspace/list.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var listOptions struct {
```

Comment:
No need for a struct here as it's not following the code style in the rest of the commands. Just `var verbose bool` is enough

Reply:
Ok fixed